scripts/k6/event-apply-race-test.js

```
@@ -0,0 +1,42 @@
+import http from 'k6/http';
+import {check, sleep} from 'k6';
+import {SharedArray} from 'k6/data';
+import {parse} from 'https://jslib.k6.io/papaparse/5.1.1/index.js';
+
+export let options = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'options' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)